linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_systemauth/rule.yml

```diff
@@ -1,18 +1,28 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019,sle12
 
 title: "Set PAM's Password Hashing Algorithm"
 
 description: |-
     The PAM system service can be configured to only store encrypted
-    representations of passwords. In <tt>/etc/pam.d/system-auth</tt>, the
+    representations of passwords. In
+    {{% if product in ["sle11", "sle12", "sle15", "opensuse"] %}}
```

Comment:
there are no `sle11` and `opensuse` products defined so it doesn't make sense to compare with them here.

Reply:
Thanks, will change to: ["sle12", "sle15"]

Reply:
PR updated to include change.